src/gui/properties/propertieswidget.ui

```diff
@@ -1017,14 +1017,14 @@
          <item>
           <widget class="QPushButton" name="selectAllButton">
            <property name="text">
-            <string>Select All</string>
+            <string>Check all items</string>
```

Comment:
>In this context Select is fine.

But don't you think `Select` could mix up with selecting list entries?
[example pic](https://cloud.githubusercontent.com/assets/9395168/15467149/8ddcae7e-210f-11e6-97ce-fc72198f2db1.png), I could select some files and press space to select/unselect them.